ui/src/components/Tile.vue

```diff
@@ -1,7 +1,15 @@
 <template>
   <div class="c-monitoror-tile" :class="classes" :style="styles">
     <div class="c-monitoror-tile--content" v-if="!isEmpty">
-      <div class="c-monitoror-tile--label">
+      <div v-if="link" class="c-monitoror-tile--label">
+        <a :href="link" target="_blank">
+          <template v-if="mergeRequestLabelPrefix">{{ mergeRequestLabelPrefix }}</template>
+          <template v-if="mergeRequestLabelPrefix && label"> @</template>
+          {{ label }}
+        </a>
+      </div>
+
+      <div v-else class="c-monitoror-tile--label">
         <template v-if="mergeRequestLabelPrefix">{{ mergeRequestLabelPrefix }}</template>
```

Comment:
I do not want clickable tiles since it is a wallboard. If we want something clickable, we need to develop a dedicated UI that makes more sense for that use.

Reply:
How about a wallboard on a touch enabled TV ?
Anyway I'll move all the "Link in title" code to a separate issue.